### DISCUSSION TITLE
PGO

### DISCUSSION THREAD

**Discussion by Imperatorn** [jinyus/related_post_gen#307] [General]
Is it ok to use PGO or not?

**Comment by zupat** [comment]
Yes, Java and Nim uses pgo

edit: go and c# uses it by default.

**Reply by Imperatorn** [reply]
Ok, D doesn't currently, so I was curious

**Reply by zupat** [reply]
It's not a requirement, it's up to the contributor to add it to their implementation.

**Comment by Imperatorn** [comment]
We're not using PGO yet, but added a SIMD-implementation. It was quite fun to do.

We will try PGO later also, but I don't know if it's a very realistic scenario for it. 
But maybe just for the lulz.
